go/cmd/vtorc/plugin_prometheusbackend.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2022 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+// This plugin imports Prometheus to allow for instrumentation
+// with the Prometheus client library
+
+import (
+	"vitess.io/vitess/go/stats/prometheusbackend"
+	"vitess.io/vitess/go/vt/servenv"
+)
+
+func init() {
+	servenv.OnRun(func() {
+		prometheusbackend.Init("vtorc")
+	})
+}
```

Comment:
@rsajwani I had to add this file like we do for `vttablet` and other binaries to register the metrics we export on the `debug/vars` page to be available on prometheus too. Thank-you for catching this.

Reply:
I don't think we have to do anything else for `Grafana`. I couldn't find anything that the other binaries were doing, so I think this is it.